docs/shared/mpas_init.yaml

```diff
@@ -0,0 +1,62 @@
+user:
+  mpas_app: /path/to/mpas_app
+platform:
+  account: me
+  scheduler: slurm
+mpas_init:
+  boundary_conditions:
+    interval_hours: 6
+    length: 6
+  execution:
+    executable: "{{ user.mpas_app }}/exec/init_atmosphere_model"
+    batchargs:
+      walltime: 01:30:00
+      cores: 4
```

Comment:
We were testing with Janet's small use case. She already had the 4-proc decomposition file, so we followed her lead. Once we nail down what the GSL group thinks is useful, we can circle back to this update.

Reply:
OK, I think I recall now that, when we were running MPAS at Spire, we used two layouts / MPI task counts: one for init and one for forecast. We found that the efficiency of init topped out at a task count much lower than forecast, but got good speedups up to that point. It might have been something like 256 tasks for init and a much larger number for forecast.